tests/affiliatedcertification/tests/affillated_certification_helm_chart.go

```diff
@@ -0,0 +1,107 @@
+package tests
+
+import (
+	"os/exec"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	tsparams "github.com/test-network-function/cnfcert-tests-verification/tests/affiliatedcertification/parameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/execute"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+)
+
+var _ = Describe("Affiliated-certification helm chart certification,", func() {
+	execute.BeforeAll(func() {
+		By("Define tnf config file")
+		err := globalhelper.DefineTnfConfig(
+			[]string{tsparams.TestHelmChartCertified},
+			[]string{tsparams.TestPodLabel},
+			[]string{},
+			[]string{})
+		Expect(err).ToNot(HaveOccurred(), "error defining tnf config file")
+		err = namespaces.Create(tsparams.TestHelmChartCertified, globalhelper.GetAPIClient())
+		Expect(err).ToNot(HaveOccurred(), "Error creating namespace")
+	})
+
+	AfterEach(func() {
+		By("remove the project")
+		err := namespaces.Clean(tsparams.TestHelmChartCertified, globalhelper.GetAPIClient())
```

Comment:
This is not deleting helm-specific resources in the namespace which is causing the second test case to fail at the "Install a helm chart" step. I think the simplest solution would be to change Clean to DeleteAndWait and move creation of the namespace into a BeforeEach 
```suggestion
		err := namespaces.DeleteAndWait(globalhelper.GetAPIClient(), tsparams.TestHelmChartCertified, tsparams.Timeout)
```